webpack.config.js

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable */
```

Comment:
All so this part, I would strongly recommend you fix it up and remove the disabling lines as this will contribute to your professionalism and gain some good ways of writing a clean code in the near future, what do you think about it? 😃

Reply:
Needs attention!